network-api/networkapi/buyersguide/templates/fragments/publication_table_of_contents.html

```diff
@@ -0,0 +1,67 @@
+<div class="container">
+  <div class="row">
+    <div class="pt-4 pt-md-5 {% if not page.notes %}pb-4 pb-md-5{% endif %} col-lg-8 offset-lg-2 justify-content-center col-12">
+      <div class="row mt-0 mt-sm-5 d-flex align-items-center">
+        <div class="col">
+          <h2>{{ page.contents_title }}</h2>
+        </div>
+        <div class="col-12">
+          {% for child in page.get_children %}
+            <div class="row publication-row pt-3 pb-sm-4 pb-0">
+              <div class="col-auto">
+                <div class="publication-chapter-number">
+                  {{ forloop.counter }}
+                </div>
+              </div>
+              <div class="col-sm col-8 ml-sm-3">
+                <h3 class="publication-chapter-title">
+                  <a href="{{ child.url }}" class="d-block mb-4">
+                    {{ child.title }}
+                  </a>
+                </h3>
+                {% if child.get_children_count %}
```

Comment:
no need for this `if` precheck: `{% for x in y %}` when `y` is an empty list means it simply doesn't generate any content.

Reply:
Correct. I originally had a container element in there. Will remove